homeassistant/components/drop_connect/binary_sensor.py

```diff
@@ -74,10 +77,23 @@ class DROPBinarySensorEntityDescription(BinarySensorEntityDescription):
         translation_key=PUMP_STATUS,
         value_fn=lambda device: device.drop_api.pump_status(),
     ),
+    DROPBinarySensorEntityDescription(
+        key=ALERT_SENSOR,
+        translation_key=ALERT_SENSOR,
+        device_class=BinarySensorDeviceClass.PROBLEM,
+        value_fn=lambda device: device.drop_api.sensor_high(),
+    ),
+    DROPBinarySensorEntityDescription(
+        key=POWER,
+        translation_key=POWER,
```

Comment:
```suggestion
        translation_key=None, # Use name provided by binary sensor device class
```

Reply:
Changed in 36db6c6ee79c627787e6a4655f6ce1d6ea67d889